packages/dart_frog_test/pubspec.yaml

```diff
@@ -0,0 +1,21 @@
+name: dart_frog_test
+description: A testing library which makes it easy to test Dart frog services
```

Comment:
**Nit:**
I think we usually add punctuation in descriptions, as in [Dart Frog](https://github.com/VeryGoodOpenSource/dart_frog/blob/023afa08a445dc1c421f484dd671da06b1ea6cbb/packages/dart_frog_cli/pubspec.yaml#L2).

```suggestion
description: A testing library which makes it easy to test Dart frog services.
```